<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HLine</class>
 <widget class="QWidget" name="HLine">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>67</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="3">
    <widget class="QLineEdit" name="le_error">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Solution</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Énoncé</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="3">
    <widget class="QLineEdit" name="le_statement">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="le_solution"/>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Erreur</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>le_solution</sender>
   <signal>textEdited(QString)</signal>
   <receiver>HLine</receiver>
   <slot>verify(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>78</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>82</x>
     <y>63</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>le_error</sender>
   <signal>textEdited(QString)</signal>
   <receiver>HLine</receiver>
   <slot>verify(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>279</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>280</x>
     <y>63</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>verify(QString)</slot>
 </slots>
</ui>
